dbbackup/utils.py

```diff
@@ -83,10 +116,12 @@ def encrypt_file(inputfile):
             always_trust = settings.GPG_ALWAYS_TRUST
             g = gnupg.GPG()
             result = g.encrypt_file(inputfile, output=filepath,
-                recipients=settings.GPG_RECIPIENT, always_trust=always_trust)
+                                    recipients=settings.GPG_RECIPIENT,
+                                    always_trust=always_trust)
             inputfile.close()
             if not result:
-                raise Exception('Encryption failed; status: %s' % result.status)
+                msg = 'Encryption failed; status: %s' % result.status
+                raise Exception(msg)
```

Comment:
Not that I want to block this or anything, but it's a bit odd to assign a variable for something used once in a function call :)